CHANGES.rst

```diff
@@ -172,6 +172,12 @@ Bug Fixes
 
 - ``astropy.utils``
 
+  - Fix to allow the C-based _fast_iterparse() VOTable XML parser to
+    relloc() its buffers instead of overflowing them.
+    (Bug occuring when Gzip-compressed VOTable XML streams are passed to
```

Comment:
@bsipocz: sorry for the confusion over parenthesis/square brackets; here is a first push with just the '`()`' dropped and '`[]`' kept.

_(Happy to also remove the whole second sentence and move the `[bug numbers]` back up higher.)_